packages/landing/src/config.map.ts

```diff
@@ -145,7 +145,11 @@ export class MapConfig extends Emitter<MapConfigEvents> {
     this.style = style ?? null;
     this.layerId = layerId.startsWith('im_') ? layerId.slice(3) : layerId;
     this.tileMatrix = tileMatrix;
-    this.labels = labels;
+    if (labels == null) {
+      this.labels = layerId === 'aerial' && !this.debug.debug;
```

Comment:
I am not a huge fan of javascripts type comparision checks eg `!foo` especially in editors where you don't know what the object is, (such as this pull request) is `debug` a string, boolean, number?

a number of options, variable naming `isSomething` implies the something is a boolean or comparing using `===`

```suggestion
      this.labels = layerId === 'aerial' && !this.isDebug;
```